entrypoint.sh

```diff
@@ -63,16 +66,16 @@ fi
 
 
 function validate_generate_tf_config() {
-    local F_PATH=$(command -v generate-tf-config)
+    F_PATH=$(command -v generate-tf-config)
```

Comment:
local variables are good...

```suggestion
    local f_path
    f_path=$(command -v generate-tf-config)
```